app-text/master-pdf-editor/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>joaompssantos@gmail.com</email>
```

Comment:
You will maintain this package. Please file a bug about this package to make sure your email in recorded in Bugzilla. Also, add the proxy-maintainer project. Developers of this project will proxy your changes.

Reply:
Should I go to bugs.gentoo.org and a new report under Gentoo Linux --> New packages?

Reply:
Yes.